rust/cjdns_sys/build.rs

```diff
@@ -0,0 +1,72 @@
+use anyhow::{bail, Result};
+use std::env;
+use std::path::{Path, PathBuf};
+use std::process::Command;
+
+fn cfiles<P: AsRef<Path>>(out: &mut Vec<PathBuf>, path: P) -> Result<()> {
```

Comment:
Consider replacing this code with an external dependency like [walkdir](https://lib.rs/crates/walkdir) or even a higher level library like [globwalk](https://lib.rs/crates/globwalk). The code will be much cleaner due to these well-designed and flexible libraries, and the hit to compile time will be very minimal (for the mentioned crates).

Reply:
Thanks for the suggestion, for now I think I'll keep it like this because it will technically be a bit faster, but if we start using these types of things in the future then I'll refactor here as well.

Reply:
Okay, that makes sense.